transformation/tests/org.jboss.tools.fuse.transformation.editor.tests.integration/build.properties

```diff
@@ -0,0 +1,5 @@
+source.. = src/main/java/,\
+           src/main/resources/
```

Comment:
Ah, this is due to github not creating empty folders.  I have it locally.  I'll delete it.